lib/Varien/Simplexml/Config.php

```diff
@@ -495,7 +495,12 @@ public function loadFile($filePath)
 
         $fileData = file_get_contents($filePath);
         $fileData = $this->processFileData($fileData);
-        return $this->loadString($fileData, $this->_elementClass);
+        $success = $this->loadString($fileData, $this-_elementClass);
```

Comment:
Shouldn't it be `$success = $this->loadString($fileData, $this->_elementClass);`, with `->`?

Reply:
You're right! I added the missing `>`